float32[] lines_start_x
float32[] lines_start_y
float32[] lines_stop_x
float32[] lines_stop_y
float32[] lines_closest_point_x
float32[] lines_closest_point_y
float32[] lines_middle_point_x
float32[] lines_middle_point_y
float32[] circles_center_x
float32[] circles_center_y
float32[] circles_radius
float32[] rectangles_corner_start_x
float32[] rectangles_corner_start_y
float32[] rectangles_corner_max_distance_x
float32[] rectangles_corner_max_distance_y
float32[] rectangles_corner_stop_x
float32[] rectangles_corner_stop_y
float32[] rectangles_corner_max_distance_mirror_x 
float32[] rectangles_corner_max_distance_mirror_y
float32[] rectangles_center_x
float32[] rectangles_center_y
